node/risk-app/server/src/js/map-page/map-controls.js

```diff
@@ -73,15 +76,15 @@ export function toggleAdvancedOptions () {
     mapControlsConsts.reservoirsContainer.classList.add(mapControlsConsts.keyAdvOptionsContainer)
     mapControlsConsts.reservoirsContainer.classList.remove(mapControlsConsts.keyContainer)
   } else {
-    if (window.location.href.includes('map=SurfaceWater')) {
+    if (currentMapPage === 'map=SurfaceWater') {
```

Comment:
Is this correct? I thought that UrlSearchParams would return just the value of the parameter, not the whole string.

Reply:
It does but there are 3 params and we only need to check which map we have entered.